docs/make.jl

```diff
@@ -3,11 +3,13 @@ using Memento
 
 makedocs(
     modules=[Memento, Memento.TestUtils],
-    format=Documenter.HTML(prettyurls=(get(ENV, "CI", nothing) == "true")),
+    format=Documenter.HTML(
+        prettyurls=get(ENV, "CI", nothing) == "true",
+        assets = ["assets/invenia.css"],
```

Comment:
```suggestion
        assets=["assets/invenia.css"],
```
Might as well fix this now

Reply:
I'm surprised I didn't notice that